migrator/src/test/java/com/google/phonenumbers/migrator/MetadataReaderTest.java

```diff
@@ -0,0 +1,14 @@
+package com.google.phonenumbers.migrator;
+
+import java.io.IOException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class MetadataReaderTest {
+
+  @Test
+  public void testInvalidFileLocation() {
+    String fileLocation = "invalid-zipfile-location";
+    Assertions.assertThrows(IOException.class, () -> new MetadataReader(fileLocation));
+  }
+}
```

Comment:
[Super nit pick] You're not putting newlines at end of file. Consider changing editor options in IntelliJ to add one during file saving (at least for Java and XML files).